Source/RuntimeImageLoader/Public/RuntimeImageLoader.h

```diff
@@ -93,10 +95,13 @@ class RUNTIMEIMAGELOADER_API URuntimeImageLoader : public UWorldSubsystem, publi
     virtual bool IsAllowedToTick() const override;
 
     URuntimeImageReader* InitializeImageReader();
+    UGIFTexture* GetOrCreateGIFTexture();
 
 private:
     UPROPERTY()
     URuntimeImageReader* ImageReader = nullptr;
+    UPROPERTY()
+    UGIFTexture* CachedGIFTexture = nullptr;
```

Comment:
Okay, let me say that again: UGifTexture is NOT a texture. It is a class that inherits from Blueprint async node, right? Where do you see the texture there?

Reply:
Resolved